### PR TITLE
Removes Nanomob

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Removes Nanomob.

- It hasn't been improved since its inception
- Most the maps are missing the friggin battle terminals which cause a debug log each round, and its such a low priority issue no one has even bothered to fix it
- The last time this was used was the 12th of January this year
- This was added during the hype of said popular mobile game, and is just old hype content

## Why It's Good For The Game
Broken code bad, missing things from maps bad, lets put it to rest.

## Images of changes
See MDB for the tiny changes, there is barely anything visible since this was missing from a lot of the maps.

## Testing
- Compiled all maps to make sure theres no remnants of this
- Tested using a PDA
- Tested using the holodeck

## Changelog
:cl: AffectedArc07
del: Removed Nanomob, you probably didnt even know it existed
/:cl:

### CODE REVIEW

File: code/modules/client/preference/loadout/loadout_general.dm [hunk 1]
```diff
@@ -162,10 +162,7 @@
 	path = /obj/item/paicard
 	cost = 2
 
-/datum/gear/mob_hunt_game
-	display_name = "Nano-Mob Hunter GO! Cartridge"
-	path = /obj/item/cartridge/mob_hunt_game
-	cost = 2
+
```
Comment (GDNgit) [inline code/modules/client/preference/loadout/loadout_general.dm:166]:
```suggestion
```

**Comment by AffectedArc07** [comment]
THIS THING USED DIRECTIONS FOR THE COLOUR STATES THIS NEEDS PURGING IN HELLFIRE
image

**Review by GDNgit — CHANGES_REQUESTED** [review]
this will cut down on like 10% of the timer usage

**Review by warriorstar-orion — APPROVED** [review]
One less subsystem good, deleting code good, less JavaScript good.

Let's Gut These ~~(not actually)~~ Mobs

**Review by DGamerL — APPROVED** [review]
Doing the thing I never had courage for

**Comment by AffectedArc07** [comment]
> this will cut down on like 10% of the timer usage

Are you serious?

**Comment by DGamerL** [comment]
> > this will cut down on like 10% of the timer usage
> 
> Are you serious?

They are, nanomobs my beloathed

**Comment by spacegeese** [comment]
noooo
th-3779027762

**Comment by Cress67** [comment]
Rest in peace, sweet nanomobs. I have never actually had a nanomob fight, but collecting them was a fun distraction as an Assistant for a brief time.

**Comment by tipoima** [comment]
I tried playing with them, never figured out how they work. 

It was also kinda strange for NT to have dedicated terminals for it, when all other games they host are interchangeable arcade cabinets.

**Pull request merged**